etc/grafana-dashboards/openstack_metrics.json

```diff
@@ -4573,11 +4708,11 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum(openstack_glance_image_bytes)",
+          "expr": "sum(openstack_glance_image_bytes) ",
```

Comment:
extraneous trailing space added?

Reply:
Not sure how those got there, came from the export I would assume o.o